components/main-chef-wrapper/main.go

```diff
@@ -47,6 +49,21 @@ func main() {
 	case "report", "capture":
 		cmd = exec.Command(dist.AnalyzeExec, allArgs...)
 
+	case "push", "push-archive":
+		cmd = exec.Command(dist.WorkstationExec, allArgs...)
```

Comment:
Let's update this so that if Rollouts are enabled but not configured correctly in validateRolloutSetup, we fail with the error message before we perform the push.   When `CHEF_AC_ROLLOUT_ENABLED` is set, the operator intent of using Rollouts is clear - so if it's not possible to use them, it should be an error.

Reply:
That sounds good. I have addressed it so we will error out and will not do the policy push if the required setup isn't complete.